action.yml

```diff
@@ -101,4 +96,4 @@ runs:
 
 branding:
   icon: 'check-circle'
-  color: 'green'
\ No newline at end of file
+  color: 'green'
```

Comment:
Is there a way to provide image for the icon?

Reply:
Added it, please take a look